engine/gamesys/src/dmsdk/gamesys/render_constants.h

```diff
@@ -152,14 +152,14 @@ namespace dmGameSystem
      * @param callback_user_data [type: void*] callback user data
      * @return result [type: dmGameObject::PropertyResult] the result
      */
-    dmGameObject::PropertyResult GetMaterialConstant(dmRender::HMaterial material, dmhash_t name_hash, dmGameObject::PropertyDesc& out_desc, bool use_value_ptr, CompGetConstantCallback callback, void* callback_user_data);
+    dmGameObject::PropertyResult GetMaterialConstant(dmRender::HMaterial material, dmhash_t name_hash, int32_t value_index, dmGameObject::PropertyDesc& out_desc, bool use_value_ptr, CompGetConstantCallback callback, void* callback_user_data);
 
     /*#
      * Used in SetMaterialConstant to set a render constant's value
      * @typedef
      * @name CompSetConstantCallback
      */
-    typedef void (*CompSetConstantCallback)(void* user_data, dmhash_t name_hash, uint32_t* element_index, const dmGameObject::PropertyVar& var);
+    typedef void (*CompSetConstantCallback)(void* user_data, dmhash_t name_hash, int32_t value_index, uint32_t* element_index, const dmGameObject::PropertyVar& var);
```

Comment:
same here

Reply:
None of the other parameters are documented though? 🤔